### ISSUE TITLE
[Bug]: TPCH load failed

### ISSUE THREAD

**Issue by zhangxu19830126** [matrixorigin/matrixone#10115]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

TPCH load failed. 

https://github.com/matrixorigin/matrixone/actions/runs/5298576569/jobs/9591070869

**Comment by zhangxu19830126** [comment]
It seems that the log node was re-scheduled. After the log node is  re-scheduled, dn fails to write the log.

**Comment by volgariver6** [comment]
还没处理这个问题

**Comment by volgariver6** [comment]
还没处理这个问题

**Comment by volgariver6** [comment]
还没处理这个问题

**Comment by volgariver6** [comment]
日志没有了。

本地环境暂时没有复现出来。

**Comment by volgariver6** [comment]
fixed by #11041
